### ISSUE TITLE
Pointed end of U+0648 in Noto Nastaliq Urdu Bold

### ISSUE THREAD

**Issue by Etewilak** [googlefonts/noto-fonts#2247]
The Arabic letter waw (U+0648) is sometimes pointed in Noto Nastaliq Urdu Bold. The following is an image of an example:
![image](https://user-images.githubusercontent.com/55674356/152951517-f6dcb684-5b74-4f1c-947f-a4fdb67caa86.png)

**Comment by simoncozens** [comment]
I agree that this is ugly, but it appears to be a design decision by Monotype. I'd like more feedback from native readers before I feel we can correct that.

Note that there is also an anchor placement issue in that glyph (WawFin.inD2):
<img width="313" alt="Screenshot 2022-02-08 at 09 04 51" src="https://user-images.githubusercontent.com/106728/152953876-b7ecc3b7-7e26-4ba3-bdb6-8de24a05d43f.png">

**Comment by Etewilak** [comment]
Other letters based on U+0648 are also affected, including U+06C4, U+0778, and U+0779.

**Comment by saadatm** [comment]
> I'd like more feedback from native readers before I feel we can correct that.

As a native Urdu reader, I am not sure why و would need such a pointed end in some sequences.

**Comment by simoncozens** [comment]
Thanks. I'll get this fixed.
